Automatic template insertion in daily notes (#195)
epwalsh/obsidian.nvim: README.md

    date_format = "%Y-%m-%d",
    -- Optional, if you want to change the date format of the default alias of daily notes.
    alias_format = "%B %-d, %Y"

>     -- Optional, if you want to automaticly insert a template from your template directory like 'daily.md'

Reviewer: ```suggestion
    -- Optional, if you want to automatically insert a template from your template directory like 'daily.md'
```

    subdir = "templates",
    date_format = "%Y-%m-%d-%a",
    time_format = "%H:%M",
	-- A map for custom variables, the key should be the variable and the value a function

> 	custom = {}

Author: I thought that are the default options. I would list it there with the empty list, just to make the exists of the option clear. Likewise, I will update it to `substitutions`.
Please correct me if I'm wrong. I thought the default config is from line 135 to 258 and on line 324 to 333 are an example. I will add a proper `substitutions` example.

> 	-- A map for custom variables, the key should be the variable and the value a function
> 	substitutions = {}

Reviewer: ```suggestion
    -- A map for custom variables, the key should be the variable and the value a function
    substitutions = {}
```

  templates = {
      subdir = "my-templates-folder",
      date_format = "%Y-%m-%d-%a",
      time_format = "%H:%M",

>       custom = {}

Reviewer: How about calling this `substitutions` (or something like that)?
```suggestion
	substitutions = {}
```

above the cursor position.

> To further customize to your personal preferences, you can supply a custom map with your on template variables.

Reviewer: How about:
```suggestion
You can also define custom template substitutions with the configuration field `templates.substitutions`. For example, to automatically substitute the template variable `{{yesterday}}` when inserting a template, you could add this to your config:
```

```lua
{

>     -- other fields ...
>    templates = {
>         substitutions = {
>             yesterday = function()
>                 return os.date("%Y-%m-%d", os.time() - 86400)
>             end
>         }
>     }
> }

Reviewer: Let's keep the indent consistent with 2 spaces:
```suggestion
-- other fields ...
templates = {
  substitutions = {
    yesterday = function()
      return os.date("%Y-%m-%d", os.time() - 86400)
    end
  }
}
```
